src/server/main_service.cc

```diff
@@ -1035,6 +1042,8 @@ std::optional<ErrorReply> Service::VerifyCommandState(const CommandId* cid, CmdA
   // If there is no connection owner, it means the command it being called
   // from another command or used internally, therefore is always permitted.
   if (dfly_cntx.conn() != nullptr && !dfly_cntx.conn()->IsPrivileged() && cid->IsRestricted()) {
+    VLOG(1) << "Non-admin attempt to execute " << cid->name() << " "
```

Comment:
we did not enable VLOG(1) in prod for main_service because of noisy VLOG(1) statements in other parts of this file. Can you please raise their level in:
1. SendMonitor
2. DispatchMonitor
3. Service::Exec
4. Service::EvalInternal